xbmc/cores/dvdplayer/DVDInputStreams/DVDInputStreamBluray.cpp

```diff
@@ -558,7 +558,7 @@ void CDVDInputStreamBluray::ProcessEvent() {
     m_player->OnDVDNavResult((void*) &pid, 3);
     break;
 
-#if (BLURAY_VERSION >= BLURAY_VERSION_CODE(0,2,2))
+#if (BLURAY_VERSION > BLURAY_VERSION_CODE(0,2,2))
```

Comment:
https://github.com/ace20022/libbluray/commit/6d88105783fa3a83963178d31f624717334ca9e0 ;) The one you're referring to is (also) emitted for bd-j menus since 0.30.